demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -8,8 +8,16 @@
 </head>
 <body>
 
-<!-- Initial meeting authentication screen with meeting and name inputs -->
+<div id="flow-fatal" class="flow">
+  <div style="vertical-align: middle; text-align: left; width: 40%; margin: 10em 30% 10em 30%;">
+  <h1>Fatal error</h1>
+  <p>An error was thrown that should not occur in ordinary use. This should be considered a fatal error for testing and canary purposes.</p>
```

Comment:
Do we want to mention the word `canary` here?

Reply:
I see no reason why not.